modin/core/io/io.py

```diff
@@ -174,42 +174,9 @@ def read_json(
         summary="Load data from Google BigQuery into query compiler",
         returns=_doc_returns_qc,
     )
-    def read_gbq(
```

Comment:
Some parameters are outdated (for example `private_key`). I guess we can just use `**kwargs`.